setup.cfg

```diff
@@ -3,6 +3,76 @@
 [bdist_rpm]
 doc-files = doc
 
+[options]
+python_requires = >=3.6
+install_requires =
+	numpy==1.16
+	scipy==1.2.0
+	pandas==0.24.0
+	scikit-learn==0.21.0
```

Comment:
Some CI jobs are failing due to the following:

>No local packages or working download links found for scikit-learn==0.21.0
>error: Could not find suitable distribution for Requirement.parse('scikit-learn==0.21.0')

Perhaps we need to change the pinned sklearn version?

Reply:
@tsalo I haven't looked in details but scikit-learn 0.21.0 is only installed under python 3.6:

https://github.com/nilearn/nilearn/blob/cc99fa76df137e9ac6010c311e15fa9ac12ce66d/.github/workflows/testing.yml#L30-L39

Reply:
That makes sense. It seems like we should at least change to 0.22 then, right?

Reply:
I think we should avoid bumping our dependencies in minor releases, but maybe we can make an exception here... (pinging @jeromedockes to see what he thinks about this).

The current CI is configured in such a way that tests are run on either:

- python 3.6 + packages pinned at minimum dependencies
- python [3.7, 3.8, 3.9] + latest versions of dependencies

which explains why we never saw the issue with python 3.6+ and sklearn==0.21.
_Note: I also realize we should add Python 3.6 to tests with latest versions._

Reply:
That's fair. Tests are currently passing, so what if I just add a comment to the description of the `min` dependency group that it should _only_ work for Python 3.6, and we can push this can down the road?

We may need a new issue for testing Python 3.6 + latest versions.